src/app/frontend/common/components/graph/component.ts

```diff
@@ -92,6 +94,15 @@ export class GraphComponent implements OnInit, OnChanges {
 
     this.yAxisSuffix_ = highestSuffix;
 
+    // Find out max value
+    this.metric.dataPoints.forEach(value => {
+      if (maxValue < value.y) {
+        maxValue = value.y;
+      }
+    });
+    // Set yScaleMax not to show area under zero
+    this.yScaleMax = maxValue ? '' : '1';
```

Comment:
It will basically always be `this.yScaleMax = '1'` as `maxValue` will always have some value if there are any points. If there are no points graphs are not displayed anyway.